### ISSUE TITLE
[SCREENREADER]: Additional Information - VoiceOver MUST set focus correctly when the component is toggled

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#8856]
- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

VoiceOver on MacOS is setting focus incorrectly when we toggle the `AdditionalInfo` component open. Animated GIF attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] VoiceOver moves focus/virtual cursor to the correct nested content when the container is expanded

## Environment

* MacOS Catalina
* Safari latest
* VoiceOver
* /disability/file-disability-claim-form-21-526ez/ (good example of the live component)

## Screenshots or Trace Logs

![voiceover-additional-info-focus-bug](https://user-images.githubusercontent.com/934879/81231344-a26c9180-8fb8-11ea-8b1c-e77a98d329e5.gif)

**Comment by rianfowler** [comment]
[Example](https://design.va.gov/components/additional-info) can be found on the design system site

**Comment by rianfowler** [comment]
step reproduce 
- turn on [voice over ](https://www.apple.com/voiceover/info/guide/_1121.html)
- use safari [here](https://design.va.gov/components/additional-info)
- follow voice over instructions to reproduce

**Comment by billfienberg** [comment]
Search results for `<AdditionalInfo`:
```
83 results - 59 files

src/applications/caregivers/components/AdditionalInfo/index.jsx:
   32:         <AdditionalInfo triggerText="What's a Primary Family Caregiver?">
   90:     <AdditionalInfo triggerText="What's a Secondary Family Caregiver?">
  172:     <AdditionalInfo triggerText="What's a Caregiver Support Coordinator ?">

src/applications/caregivers/definitions/caregiverUI.js:
  173:           <AdditionalInfo triggerText="Learn more about Medicaid">
  185:           <AdditionalInfo triggerText="Learn more about Medicare">
  199:           <AdditionalInfo triggerText="Learn more about Tricare">
  211:           <AdditionalInfo triggerText="Learn more about CHAMPVA">

src/applications/disability-benefits/2346/components/Accessories.jsx:
  214:           <AdditionalInfo triggerText="What if I don't see the accessories I need?">

src/applications/disability-benefits/2346/components/Batteries.jsx:
  231:           <AdditionalInfo triggerText="What if I don't see my device?">

src/applications/disability-benefits/2346/schemas/address-schema.js:
  59:     <AdditionalInfo

src/applications/disability-benefits/526EZ/helpers.jsx:
  689:     <AdditionalInfo triggerText="What does this mean?">
  815:   <AdditionalInfo triggerText="Read the full text.">
  922:   <AdditionalInfo triggerText="What does this mean?">

src/applications/disability-benefits/686/helpers.jsx:
  135:     <AdditionalInfo triggerText="What does this mean?">

src/applications/disability-benefits/686c-674/config/address-schema.js:
  62:     <AdditionalInfo

src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-place-of-birth/childStatusDescription.jsx:
  14:     <AdditionalInfo triggerText="What are some examples of additional evidence?">

src/applications/disability-benefits/686c-674/config/chapters/report-add-a-spouse/current-marriage-information/helpers.jsx:
  14:     <AdditionalInfo triggerText="What are some examples of additional evidence?">

src/applications/disability-benefits/996/content/contestedIssues.jsx:
  46:     <AdditionalInfo triggerText="Don’t see the issue you’re looking for?">

src/applications/disability-benefits/all-claims/containers/FormSavedPage.jsx:
  19:       <AdditionalInfo triggerText="What is an Intent to File?">

src/applications/disability-benefits/all-claims/content/addDisabilities.jsx:
  17:     <AdditionalInfo triggerText="What if I don’t know the name of my condition?">

src/applications/disability-benefits/all-claims/content/evidenceTypes.jsx:
  5:   <AdditionalInfo triggerText="Which evidence type should I choose?">

src/applications/disability-benefits/all-claims/content/incidentDate.jsx:
  27:       <AdditionalInfo

src/applications/disability-benefits/all-claims/content/incomeDetails.jsx:
  22:     <AdditionalInfo

src/applications/disability-benefits/all-claims/content/itfWrapper.jsx:
  32:   <AdditionalInfo

src/applications/disability-benefits/all-claims/content/privateMedicalRecords.jsx:
  13:     <AdditionalInfo
  39:     <AdditionalInfo

src/applications/disability-benefits/all-claims/content/privateMedicalRecordsRelease.jsx:
  19:   <AdditionalInfo

src/applications/disability-benefits/all-claims/content/ptsdTypeInfo.jsx:
  29:   <AdditionalInfo

src/applications/disability-benefits/all-claims/content/ptsdWalkthroughChoice.jsx:
  23:     <AdditionalInfo

src/applications/disability-benefits/all-claims/content/recentEarnedIncome.jsx:
  6:   <AdditionalInfo triggerText="What’s gross income?">

src/applications/disability-benefits/all-claims/content/recentJobApplications.jsx:
  18:   <AdditionalInfo triggerText="What’s substantially gainful employment?">

src/applications/disability-benefits/all-claims/content/secondaryOtherSources.jsx:
  23:   <AdditionalInfo

src/applications/disability-benefits/all-claims/content/unemployabilityDates.jsx:
  25:     <AdditionalInfo

src/applications/disability-benefits/all-claims/content/unemployabilityDisabilities.jsx:
  34:       <AdditionalInfo

src/applications/disability-benefits/all-claims/pages/contactInformation.js:
  93:           <AdditionalInfo

src/applications/disability-benefits/all-claims/pages/supplementalBenefits.jsx:
  16:   <AdditionalInfo triggerText="How do these benefits affect my claim?">

src/applications/disability-benefits/all-claims/pages/terminallyIll.js:
  8:   <AdditionalInfo triggerText="Why does this matter?">

src/applications/disability-benefits/bdd/content/evidenceTypes.jsx:
  5:   <AdditionalInfo triggerText="Which evidence type should I choose?">

src/applications/disability-benefits/bdd/content/militaryHistory.jsx:
  9:     <AdditionalInfo triggerText="What's this?">

src/applications/edu-benefits/0994/content/bankInformation.jsx:
  30:   <AdditionalInfo

src/applications/gi/components/profile/InstitutionProfile.jsx:
  101:               <AdditionalInformation

src/applications/gi/tests/components/AdditionalInformation.unit.spec.jsx:
   8: describe('<AdditionalInformation>', () => {
  17:     const wrapper = shallow(<AdditionalInformation {...defaultProps} />);
  33:       const wrapper = shallow(<AdditionalInformation {...props} />);
  47:       const wrapper = shallow(<AdditionalInformation {...ojtProps} />);
  61:       const wrapper = shallow(<AdditionalInformation {...noMessageProps} />);
  67:       const wrapper = mount(<AdditionalInformation {...props} />);

src/applications/hca/helpers.jsx:
  434:       <AdditionalInfo triggerText="What if my expenses are higher than my annual income?">
  447:       <AdditionalInfo triggerText="Learn more about minimum essential coverage.">
  459:       <AdditionalInfo triggerText="Learn more about Medicaid.">
  469:       <AdditionalInfo triggerText="Learn more about Medicare Part A insurance.">

src/applications/personalization/account/components/LoginSettings.jsx:
  42:         <AdditionalInfo triggerText="What is ID.me?">

src/applications/personalization/preferences/components/PreferenceItem.jsx:
  51:       <AdditionalInfo

src/applications/personalization/profile-2/components/IdentityNotVerified.jsx:
  48:         <AdditionalInfo triggerText="How will VA.gov verify my identity?">

src/applications/personalization/profile-2/components/MilitaryInformation.jsx:
  104:         <AdditionalInfo

src/applications/personalization/profile-2/components/connected-apps/ConnectedApp.jsx:
  71:           <AdditionalInfo triggerText={`Learn about ${title}`}>

src/applications/personalization/profile-2/components/direct-deposit/DirectDepositContent.jsx:
  168:         <AdditionalInfo triggerText="Where can I find these numbers?">

src/applications/personalization/profile-2/components/personal-information/ContactInformationSection.jsx:
  16:     <AdditionalInfo triggerText="Which of my benefits will use this contact information?">

src/applications/personalization/profile-2/components/personal-information/GenderAndDOBSection.jsx:
  31:     <AdditionalInfo triggerText="How do I update my personal information?">

src/applications/personalization/profile360/components/ContactInformationExplanation.jsx:
  23:       <AdditionalInfo

src/applications/personalization/profile360/components/IdentityVerification.jsx:
  25:           <AdditionalInfo triggerText={`How will VA.gov verify my identity?`}>

src/applications/personalization/profile360/components/MilitaryInformation.jsx:
  99:         <AdditionalInfo

src/applications/personalization/profile360/components/PersonalInformation.jsx:
  54:         <AdditionalInfo

src/applications/personalization/profile360/containers/PaymentInformation.jsx:
   43:       <AdditionalInfo
   72:     <AdditionalInfo
  256:           <AdditionalInfos recordProfileNavEvent={recordProfileNavEvent} />

src/applications/pre-need/components/SupportingDocumentsDescription.jsx:
  54:       <AdditionalInfo triggerText="What kinds of documents should I provide?">
  78:       <AdditionalInfo triggerText="Can I mail or fax documents?">

src/applications/vaos/components/AppointmentRequestListItem.jsx:
  124:           <AdditionalInfoRow

src/applications/vaos/components/ConfirmationRequestInfo.jsx:
  155:           <AdditionalInfo

src/applications/vaos/components/ConfirmedAppointmentListItem.jsx:
  156:                 <AdditionalInfoRow

src/applications/vaos/containers/PreferredDatePage.jsx:
  71:             <AdditionalInfo triggerText="Why are you asking me this?">

src/platform/site-wide/cta-widget/components/messages/DeactivatedMHVIds.jsx:
  18:         <AdditionalInfo triggerText="Call the My HealtheVet help desk">
  31:           <AdditionalInfo triggerText="Or submit an online help request to My HealtheVet">

src/platform/site-wide/cta-widget/components/messages/MultipleIds.jsx:
  14:         <AdditionalInfo triggerText="Call the My HealtheVet help desk">
  27:           <AdditionalInfo triggerText="Or submit an online help request to My HealtheVet">

src/platform/site-wide/cta-widget/components/messages/NeedsSSNResolution.jsx:
  23:         <AdditionalInfo triggerText="Call the VA benefits hotline">
  51:           <AdditionalInfo triggerText="Or ask us a question online">

src/platform/site-wide/cta-widget/components/messages/RegisterFailed.jsx:
  29:         <AdditionalInfo triggerText="Call the My HealtheVet help desk">
  42:           <AdditionalInfo triggerText="Or submit an online help request to My HealtheVet">

src/platform/site-wide/cta-widget/components/messages/UpgradeFailed.jsx:
  27:         <AdditionalInfo triggerText="Call the My HealtheVet help desk">
  40:           <AdditionalInfo triggerText="Or submit an online help request to My HealtheVet">

src/platform/user/profile/vet360/components/AddressField/address-schemas.js:
  88:         <AdditionalInfo
```
